src/model/reviews.model.js

```diff
@@ -0,0 +1,41 @@
+import { Schema, model } from "mongoose";
+
+const reviewSchema = new Schema({
+  comment: {
+    type: String,
+    required: [true, "El comentario es requerido"],
+  },
+  rating_cleanliness: {
+    type: int,
+    required: [true, "La calificación de limpieza es requerida"],
+  },
+  rating_staff: {
+    type: int,
+    required: [true, "La calificación del personal es requerida"],
+  },
+  rating_facilities: {
+    type: int,
+    required: [true, "La calificación de las instalaciones es requerida"],
+  },
+  tp_status: {
+    type: String,
+    required: [true, "El estado es requerido"],
+    enum: ["ACTIVE", "INACTIVE"],
+  },
+  is_custom: {
+    type: Boolean,
+    required: [true, "Se requiere indicar si es personalizado"],
+  },
```

Comment:
esto esta mal. deberia ser, `is_customer` para validar si el que esta haciendo el comentario fue un usuario de el hotel segun nuestra data o si es un maje x, haciendo review de un servicio que no uso.

```suggestion
  is_customer: {
    type: Boolean,
    required: [true, "Se requiere indicar fue cliente"],
  },
```